sound/soc/intel/common/soc-acpi-intel-mtl-match.c

```diff
@@ -441,6 +501,12 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_mtl_sdw_machines[] = {
 		.drv_name = "sof_sdw",
 		.sof_tplg_filename = "sof-mtl-rt715-rt711-rt1308-mono.tplg",
 	},
+	{
+		.link_mask = GENMASK(3, 0),
+		.links = mtl_rt713_l0_rt1316_l12_rt1713_l3,
+		.drv_name = "sof_sdw",
+		.sof_tplg_filename = "sof-mtl-rt713-l0-rt1316-l12-rt1713-l3.tplg",
+	},
```

Comment:
isn't this in https://github.com/thesofproject/linux/pull/4538 already?

Reply:
Hm... yes not sure what has happened there apologies let me repush.

Reply:
Double hmm... github is weird. The patch extra patch here is merged on the sof branch and has the same SHA as it does here, so I blatantly got it from the remote when I rebased this morning. No idea why github would include it in this review. Repushing does nothing because its just the same as the code that exists in the repo. I guess I will make a whitespace change in my patch or something and repush in the hope that enables GitHub to sort its non-sense out. If that doesn't work I will close this review and open a new one.